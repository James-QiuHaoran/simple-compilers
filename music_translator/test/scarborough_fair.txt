l-lM-Mt>Dtl--
0MSL-SM/FRM--
L-LS-MMRDts--
l-MR-Dtlsl--
