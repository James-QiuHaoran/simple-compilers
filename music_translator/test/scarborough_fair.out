E2EB2BF>GFE3zBde2dB^cAB3e2ed2BBAGFD3E2BA2GFEDE3

